vision_agent/tools/tools_v2.py

```diff
@@ -167,15 +326,104 @@ def display_bounding_boxes(
     return np.array(pil_image.convert("RGB"))
 
 
-def get_tool_documentation(funcs: List[Callable]) -> str:
+def display_segmentation_masks(
```

Comment:
nit, naming: how about `overlay_segmentation_masks`? 
it's not really displaying the mask...

```suggestion
def overlay_segmentation_masks(
```

Reply:
Good point, I'll change that